cassandra/cluster.py

```diff
@@ -1235,7 +1235,7 @@ def shutdown(self):
 
         self.control_connection.shutdown()
 
-        for session in self.sessions:
+        for session in tuple(self.sessions):
```

Comment:
I wouldn't replace it here since I think adding/removing a session at the same time we call `cluster.shutdown` is a bug of the application, not of the driver